### PR TITLE
Delete

### PR DESCRIPTION
**Pull request by wattli**

### CODE REVIEW

File: _docs/setup/kubernetes/quick-start.md [hunk 1]
```diff
@@ -244,6 +244,13 @@ For example, run the following command on a MacOS or Linux system:
 1.  Install Istio's core components. Choose one of the two _**mutually exclusive**_ options below or alternately install
 with the [Helm Chart]({{home}}/docs/setup/kubernetes/helm-install.html):
 
+    If you already have Istio installed with version 0.7 or older, please delete the legacy service and deployments by running
+        ```command
+        $ kubectl delete service istio-mixer -n istio-system
+        $ kubectl delete deploy istio-mixer istio-ca -n istio-system
+
```
Comment (geeknoid) [inline _docs/setup/kubernetes/quick-start.md]: Remove blank line.

Comment (wattli) [reply _docs/setup/kubernetes/quick-start.md]: done

**Comment by costinm** [comment]
Can we instead delete or disable  automatically ? There are upgrade cases for GCP, etc that don't have manual steps.

You can either create a deployment with the same name and no action, or use the post-install to delete the pod.

**Comment by wattli** [comment]
Empty deploy seems hacky. Post install seems interesting, let's talk tomorrow :)

**Pull request merged**